scripts/qa_crowbarsetup.sh

```diff
@@ -3608,10 +3608,23 @@ function oncontroller_run_tempest
     pushd /var/lib/openstack-tempest-test
     sysctl -e kernel.sysrq=1 net.ipv4.neigh.default.gc_thresh1=0
     local tempestret
+    local blacklist=""
+
+    if iscloudver 9; then
+
+        blacklistfile=/var/lib/openstack-tempest-test/blacklist.txt
+        blacklist="--blacklist-file $blacklistfile"
+        cat -  << EOF
+id-f5dfcc22-45fd-409f-954c-5bd500d7890b
+id-301f5a30-1c6f-4ea0-be1a-91fd28d44354
+id-bdbb5441-9204-419d-a225-b4fdbfb1a1a8
+id-6bba729b-3fb6-494b-9e1e-82bbd89a1045
+EOF
+    fi
```

Comment:
I think you could use the test name itself like `tempest.scenario.test_network_basic_ops.TestNetworkBasicOps.test_update_instance_port_admin_state` which would make this more readable

Reply:
I was considering that, but decided against it because occassionally tests are renamed and the id is more unique in that regard. you can still use it as any other identifier to tempest run -r and afaik it will report "unexpected passes"  as well at teh end so we can remove it. I don't want this list to grow or stay around for too long. Just want to avoid the impression that "its red and always been red so I don't care"

Reply:
Even so, could you at least add the test name as a comment on top of each ID, even if this is temporary ? `qa_crowbarsetup.sh` is already sprinkled with enough cryptic exceptions as is.

Reply:
In that case we should document why we are removing each of those, so that when <reason> is not valid anymore, we can remove the test from the blacklist...

Reply:
Or create an SCRD ticket documenting the reason that we are temporarily blacklisting the tests, with details/explanation for each, and include the SCRD reference in the script?

Reply:
> Even so, could you at least add the test name as a comment on top of each ID, even if this is temporary ? `qa_crowbarsetup.sh` is already sprinkled with enough cryptic exceptions as is.

The name doesn't really bothers me, as you can find that in the commit message by doing a git blame. But the reason is obscure, so there is no reason to believe it will ever be removed.

Reply:
what do you mean by obscure? they fail, they always have failed, and its likely because mkcloud does not setup enough vlans so that they would ever pass. all of them can be fixed in one go.